### DISCUSSION TITLE
Investigating Closed vs. Open Source

### DISCUSSION THREAD

**Discussion by avaitt** [albertkun/21S-ASIAAM-191A#109] [Past Thinking Caps]
The issue of closed vs open-source systems is multi-faceted. Open source has provided us with a myriad of tools and technologies that are often free to use, making our lives significantly easier. However, nothing is truly free, and rather than a monetary fee, our data has turned into a commodity that fuels a multi-billion dollar industry. At the end of the day, open-source pits convenience against privacy, security, and morality.  Conversely, closed source can inhibit growth especially for the purposes of scling. When we are forced to pay exponentially more for services that are slightly better than a free tier, there is a clear abuse of power.

However, this added cost comes with the benefit of more up-to-date software maintenance, often greater documentation, and a system that largely does not abuse the privacy of customers. The issue of closed vs open-source software and technology is not black and white. If we were to shift exclusively to one or another, the consequences would be disastrous. In order to create an affordable, middle-way we ought to focus on what we value the most in our products (affordability or privacy).

**Comment by jason-muljadi** [comment]
It seems as if it is always hard to achieve that balance. When corporations can keep hiring the best and brightest programmers for a profit in this profit-driven world, morality and influence can be shifted to the conglomeracy. Though ethics board exist and there are always watchdogs that disallow the continuous abuses of power (not to mention that some companies do have the ability to do good), it is hard for there to achieve this balance of affordability and privacy.
